pkg/daemons/control/deps/deps.go

```diff
@@ -31,7 +31,8 @@ const (
 	ipsecTokenSize = 48
 	aescbcKeySize  = 32
 
-	RequestHeaderCN = "system:auth-proxy"
+	RequestHeaderCN      = "system:auth-proxy"
+	EncryptionStartState = "start"
```

Comment:
Where is this used?

Reply:
Its not! I stripped out the use at some point. Good catch :)